integration/spark2/src/test/scala/org/apache/carbondata/spark/testsuite/booleantype/BooleanDataTypesInsertTest.scala

```diff
@@ -945,4 +948,41 @@ class BooleanDataTypesInsertTest extends QueryTest with BeforeAndAfterEach with
     )
   }
 
+  test("Inserting table with bad records, and SORT_COLUMNS is boolean column") {
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.ENABLE_AUTO_LOAD_MERGE, "true")
```

Comment:
How is this test related to comments?

Reply:
this test is not related to col comment, this test is just added in this PR which is for boolean datatype